laser_slam/src/benchmarker.cpp

```diff
@@ -0,0 +1,107 @@
+
+#include <chrono>
+#include <cmath>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <mutex>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include <glog/logging.h>
+
+#include "laser_slam/benchmarker.hpp"
+
+namespace laser_slam {
+
+void Benchmarker::addMeasurement(
+    const std::string& name,
+    const std::chrono::time_point<clock>& start,
+    const std::chrono::time_point<clock>& end) {
+
+  constexpr double mus_to_ms = 1.0 / 1000.0;
```

Comment:
Three different approaches leading to the same (optimal) result: https://godbolt.org/g/LqsSmi
I personally prefer this version in this case for readability :)